qcodes/instrument_drivers/AlazarTech/utils.py

```diff
@@ -1,12 +1,24 @@
-import warnings
+"""
+The module provides useful utility objects for the Alazar driver class.
+The purpose of these is more to provide relevant functionality and less to
+improve the code structure (which is more the purpose of the 
+:mod:`.AlazarTech.helpers` module).
+"""
+
 
 from qcodes.instrument.parameter import Parameter
-from qcodes.utils import validators
+
 
 class TraceParameter(Parameter):
     """
     A parameter that keeps track of if its value has been synced to
-    the Instrument by setting a flag when changed.
+    the ``Instrument``. To achieve that, this parameter sets
+    the ``_parameters_synced`` attribute of the ``Instrument`` to ``False``
+    when the value of the parameter is changed.
+
+    This parameter is useful for the Alazar card driver. Syncing parameters to
+    an Alazar card is not instance, hence it makes sense to first set the
```

Comment:
Syncing parameters to an Alazar card is slow?

Reply:
yes, i'll fix the mistyping.